hilti/runtime/src/util.cc

```diff
@@ -400,6 +400,18 @@ hilti::rt::ByteOrder hilti::rt::systemByteOrder() {
 #endif
 }
 
+std::string hilti::rt::detail::adl::to_string(const hilti::rt::ByteOrder& x, tag /*unused*/) {
+    switch ( x ) {
+        case hilti::rt::ByteOrder::Little: return "Little";
+        case hilti::rt::ByteOrder::Big: return "Big";
+        case hilti::rt::ByteOrder::Network: return "Network";
+        case hilti::rt::ByteOrder::Host: return "Host";
+        case hilti::rt::ByteOrder::Undef: return "Undef";
```

Comment:
Enum rendering should include the type name:

```suggestion
        case hilti::rt::ByteOrder::Little: return "ByteOrder::Little";
        case hilti::rt::ByteOrder::Big: return "ByteOrder::Big";
        case hilti::rt::ByteOrder::Network: return "ByteOrder::Network";
        case hilti::rt::ByteOrder::Host: return "ByteOrder::Host";
        case hilti::rt::ByteOrder::Undef: return "ByteOrder::Undef";
```